api/lib/notifications.js

```diff
@@ -105,6 +105,10 @@ function getMessageTitle(networks, notification) {
 
     case eventTypes.LUNIE_UPDATE:
       return data.title
+    case eventTypes.SLASH:
+      return `Validator ${data.operatorAddress} got slashed ${data.amount}.`
```

Comment:
will this be a fiat amount or token amount?

Reply:
right now it is an onchain amount, need to adjust to show as coin value. should we show as fiat value? but which fiat value? so I guess we show the LunieCoin value

Reply:
one day, users should be able to choose their preferred currency. for now though, if it's token amount it should include the denom and not the `$`.

Reply:
see update

Reply:
i don't see what changed with regard to fiat / token amounts

Reply:
was missing a push